api/src/controllers/versions.ts

```diff
@@ -211,9 +211,15 @@ router.get(
 
 		const { outdated, mainHash } = await service.verifyHash(version['collection'], version['item'], version['hash']);
 
-		const saves = await service.getVersionSavesById(version['id']);
+		let current;
 
-		const current = assign({}, ...saves);
+		if (version['delta']) {
+			current = version['delta'];
+		} else {
+			const saves = await service.getVersionSavesById(version['id']);
+
+			current = assign({}, ...saves);
```

Comment:
Should we consider deleting the old versions once they have been migrated and saved here in the fallback?

Reply:
The old versions are `revisions` and they are there as a log.